### DISCUSSION TITLE
Script to use docker image to build from outside ORFS/flow folder and run GUI

### DISCUSSION THREAD

**Discussion by oharboe** [The-OpenROAD-Project/OpenROAD-flow-scripts#1092] [General]
I created a pull request https://github.com/The-OpenROAD-Project/OpenROAD-flow-scripts/pull/1091 with a script that I find useful.

It allows us to use the docker image to build source code in our own private repository and to run the GUI from within the Docker image.

The Docker image is a nice way to deploy updates and to lock down the version of ORFS and OpenROAD to a git commit in our own repository.

For instance, we can upgrade to the latest version of ORFS and OpenROAD with a pull request against our own private repository and run all regression tests automatically before merging an upgrade.

Thoughts?

**Comment by rovinski** [comment]
I think that contributed helper scripts should probably not go in the top-level directory. It can get quite cluttered there. Maybe in `/etc` or `/flow/util`. Also I think `or` isn't a very descriptive name of what the script does. Maybe `docker_shell` or similar.
